sdk/storage/README.md

```diff
@@ -8,9 +8,9 @@ This project provides client libraries in JavaScript that makes it easy to consu
 - [Source Code - File](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-share)
 - [Source Code - Queue](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-queue)
 - [Product documentation](https://docs.microsoft.com/en-us/azure/storage)
-- @azure/storage-blob [Package (npm)](https://www.npmjs.com/package/@azure/storage-blob/v/12.0.0-preview.5)
-- @azure/storage-file-share [Package (npm)](https://www.npmjs.com/package/@azure/storage-file-share/v/12.0.0-preview.5)
-- @azure/storage-queue [Package (npm)](https://www.npmjs.com/package/@azure/storage-queue/v/12.0.0-preview.5)
+- @azure/storage-blob [Package (npm)](https://www.npmjs.com/package/@azure/storage-blob)
+- @azure/storage-file-share [Package (npm)](https://www.npmjs.com/package/@azure/storage-file-share/v/12.0.0-preview.6)
+- @azure/storage-queue [Package (npm)](https://www.npmjs.com/package/@azure/storage-queue
```

Comment:
```suggestion
- @azure/storage-queue [Package (npm)](https://www.npmjs.com/package/@azure/storage-queue)
```

Reply:
I have already made this change

Reply:
No missed a `)` bracket at the end of the line. :)